tests/test_track_objects_block.py

```diff
@@ -12,7 +10,7 @@ class TestTrackObjects(NIOBlockTestCase):
     def setUp(self):
         super().setUp()
         sys.modules['cv2'] = MagicMock()
-        sys.modules['object_tracker'] = MagicMock()
+        sys.modules['imutils'] = MagicMock()
```

Comment:
I had to adjust this to get the tests to run locally on my machine, if you don't think this is correct feel free to ignore